site/src/pages/HealthPage/ProvisionerDaemonsPage.tsx

```diff
@@ -0,0 +1,165 @@
+                  gap: 24,
+                }}
+              >
+                <div
+                  css={{
+                    display: "flex",
+                    alignItems: "center",
+                    gap: 24,
+                    objectFit: "fill",
+                  }}
+                >
+                  <div css={{ lineHeight: "160%" }}>
+                    <h4 css={{ fontWeight: 500, margin: 0 }}>
+                      {provisioner_daemon.name}
+                    </h4>
+                    <span css={{ color: theme.palette.text.secondary }}>
+                      <code>{provisioner_daemon.version}</code>
+                    </span>
+                  </div>
+                </div>
+                <div
+                  css={{
+                    marginLeft: "auto",
+                    display: "flex",
+                    flexWrap: "wrap",
+                    gap: 12,
+                  }}
+                >
+                  <Tooltip title="API Version">
+                    <Pill icon={<SwapHoriz />}>
+                      <code>{provisioner_daemon.api_version}</code>
+                    </Pill>
+                  </Tooltip>
+                  <Tooltip title="Scope">
+                    <Pill icon={iconScope}>{titleCase(daemonScope)}</Pill>
+                  </Tooltip>
+                  {Object.keys(extraTags).map((k) => (
+                    <Tooltip key={k} title={k}>
+                      <Pill key={k} icon={<Sell />}>
+                        {extraTags[k]}
+                      </Pill>
+                    </Tooltip>
+                  ))}
+                </div>
+              </header>
+
+              <div
+                css={{
+                  borderTop: `1px solid ${theme.palette.divider}`,
+                  display: "flex",
+                  alignItems: "center",
+                  justifyContent: "space-between",
+                  padding: "8px 24px",
+                  fontSize: 12,
+                  color: theme.palette.text.secondary,
+                }}
+              >
+                {warnings.length > 0 ? (
+                  <div css={{ display: "flex", flexDirection: "column" }}>
+                    {warnings.map((warning, i) => (
+                      <span key={i}>{warning.message}</span>
+                    ))}
+                  </div>
+                ) : (
+                  <span>No warnings</span>
+                )}
+                {provisioner_daemon.last_seen_at && (
+                  <span
+                    css={{ color: theme.palette.text.secondary }}
+                    data-chromatic="ignore"
+                  >
+                    Last seen {createDayString(provisioner_daemon.last_seen_at)}
+                  </span>
+                )}
+              </div>
+            </div>
+          );
+        })}
+      </Main>
+    </>
+  );
+};
+
+const titleCase = (s: string): string => {
+  return s.charAt(0).toLocaleUpperCase() + s.slice(1);
```

Comment:
We don't need to check that `s` is not empty accidentally, right?

Reply:
Honestly I'd rather not need to have this function at all, but I couldn't find an existing one in the codebase. I didn't look too hard though. Maybe @BrunoQuaresma can point me towards a better alternative?

Reply:
You can do it using CSS. 
```css
&:first-letter: { text-transform: "uppercase" }
```
PS: It needs to be a block or flex element.